ui/raidboss/data/06-ew/raid/p7s.ts

```diff
@@ -147,6 +153,120 @@ const triggerSet: TriggerSet<Data> = {
         return { infoText: output.spreadThenStack!() };
       },
     },
+    {
+      id: 'P7S Bull and Minotaur Tethers',
+      // 0006 Immature Io (Bull) Tether
+      // 0039 Immature Minotaur Tether
+      // Forbidden Fruit 4: 4 Bull Tethers, 2 Minotaur Tethers, 1 Non-tethered Minotaur
+      // Famine: 4 Minotaur Tethers, 2 Non-tethered Minotaurs, 2 Static Birds
+      // Death: 2 Bulls with Tethers, 1 Bull casting Puddle AoE, 2 Static Birds
+      // War: 4 Bull Tethers, 2 Minotaur Tethers, 2 Bird Tethers
+      // TODO: Get locations with OverlayPlugin via X, Y and bird headings?
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: ['0006', '0039'] }),
+      condition: (data) => !data.stopTethers,
+      preRun: (data, matches) => data.tetherCollect.push(matches.target),
+      delaySeconds: 0.1,
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          bullTether: {
+            en: 'Bull Tether ${location}',
+          },
+          deathBullTether: {
+            en: 'Bull Tether ${location}',
+          },
+          warBullTether: {
+            en: 'Bull Tether ${location}',
+          },
+          minotaurTether: {
+            en: 'Minotaur Tether ${location}',
+          },
+          famineMinotaurTethers: {
+            en: 'Cross Minotaur Tethers ${location}',
+          },
+          warMinotaurTethers: {
+            en: 'Minotaur Tether ${location}',
+          },
+          noTether: {
+            en: 'No Tether, Bait Minotaur Cleave ${location}',
+          },
+          famineNoTether: {
+            en: 'No Tether, Bait Minotaur Cleave ${location}',
+          },
+          middle: {
+            en: '(Middle)',
+          },
+          lineAoE: {
+            en: '(Line AoE)',
+          },
+          bigCleave: {
+            en: '(Big Cleave)',
+          },
+        };
+
+        if (data.me === matches.target) {
+          // Bull Tethers
+          if (matches.id === '0006') {
+            if (data.tetherCollectPhase === 'death')
+              return { infoText: output.deathBullTether!({ location: '' }) };
+            if (data.tetherCollectPhase === 'war')
+              return { infoText: output.warBullTether!({ location: '' }) };
+            return { infoText: output.bullTether!({ location: output.lineAoE!() }) };
```

Comment:
This isn't really `location`, right? as all bull tethers are line aoes?  Can this just be in the trigger itself, and maybe drop `location: ''` from things that aren't using it?

Reply:
Sounds good, I'll see about a future pr to maybe have the location of the entity added. I have the code to get the location of the entity, but to get the one you are tethered to still would need to be done so it would take more time.